### PR TITLE
Jxiao/add decimal fn

### PR DESCRIPTION
**Pull request by jarry-xiao**

### CODE REVIEW

File: typescript/phoenix-sdk/src/market.ts [hunk 1]
```diff
@@ -245,4 +246,23 @@ export class Market {
       inAmount,
     });
   }
+
+  getPriceDeicmalPlaces(): number {
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/market.ts:250]: typo in name

where is this used?

File: typescript/phoenix-sdk/src/market.ts [hunk 2]
```diff
@@ -245,4 +246,23 @@ export class Market {
...
+      target /= 2;
+      exp2 += 1;
+    }
+    let exp5 = 0;
+    while (target % 5 === 0) {
+      target /= 5;
+      exp5 += 1;
+    }
+    let precision = Math.max(exp2, exp5);
+    return precision === 0 ? 3 : precision;
```
Comment (eugene-chen) [inline typescript/phoenix-sdk/src/market.ts:266]: perhaps `return Math.max(3, precision)`?

File: typescript/phoenix-sdk/src/market.ts (hunk 1 above)
Comment (jarry-xiao) [reply typescript/phoenix-sdk/src/market.ts:250]: good call, Logan recommended that this was a better place for it (for used in data ui)

**Review by loganjross — APPROVED** [review]
other than typo looks good to me!

File: typescript/phoenix-sdk/src/market.ts (hunk 2 above)
Comment (jarry-xiao) [reply typescript/phoenix-sdk/src/market.ts]: Think there's some additional work related to the `raw_base_units_per_base_unit` factor... will fix

**Pull request merged**